exercises/Scientific-2-Python.md

```diff
@@ -1,10 +1,11 @@
 ---
 layout: exercise
 title: Scientific 2
+subtitle: Mammal Bosy Size Pt 2
```

Comment:
> Bosy -> Body

Done. Not sure where my head was for this one. Also fixed the second link.

On Mon, Jul 20, 2015 at 5:32 AM, Ethan White <notifications@github.com>
wrote:

> In exercises/Scientific-2-Python.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/149#discussion_r34982835>
> :
>
> > @@ -1,10 +1,11 @@
> >  ---
> >  layout: exercise
> >  title: Scientific 2
> > +subtitle: Mammal Bosy Size Pt 2
>
> "Bosy" -> "Body"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/149/files#r34982835>
> .
>